service/reducers/entities/helpers.js

```diff
@@ -30,35 +32,17 @@ function makePostListNonNull(pl) {
     return postList;
 }
 
-export function addPosts(state, action) {
-    const newState = {...state};
+function storePosts(state, action) {
     const newPosts = action.data;
-    const id = action.channel_id;
+    const channelId = action.channelId;
 
     if (isPostListNull(newPosts)) {
         return state;
     }
 
-    if (action.checkLatest) {
-        const currentLatest = newState.latestPageTime[id] || 0;
-        if (newPosts.order.length >= 1) {
-            const newLatest = newPosts.posts[newPosts.order[0]].create_at || 0;
-            if (newLatest > currentLatest) {
-                newState.latestPageTime = {
-                    ...newState.latestPageTime,
-                    [id]: newLatest
-                };
-            }
-        } else if (currentLatest === 0) {
-            // Mark that an empty page was received
-            newState.latestPageTime = {
-                ...newState.latestPageTime,
-                [id]: 1
-            };
-        }
-    }
-
-    const combinedPosts = {...makePostListNonNull(newState.postsInfo[id].postList)};
+    const postInfo = state[channelId];
+    const postList = postInfo || {};
```

Comment:
Unless you need both `postInfo` somewhere else, can you maybe combine these two lines? It's kind of confusing to me to have two differently named variables with basically the same data